WTWSMS/common/governments/WtWSMS_feudal_governments.txt

```diff
@@ -140,13 +141,6 @@ feudal_governments = {
 			bureaucratic_government
 		}
 		potential = {
-			NAND = {
-				controls_religion = yes
-				OR = {
-					religion_group = christian
-					religion_group = pagan
-				}
-			}
```

Comment:
What do you mean? The idea of preventing a religious head from going Bureaucratic has nothing to do with Judaism.